app/src/main/java/com/example/taskyy/data/local/room_entity/agenda_entities/PendingEventRetryEntity.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.taskyy.data.local.room_entity.agenda_entities
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.example.taskyy.ui.enums.AgendaItemAction
+
+@Entity(tableName = "pending_event_retry_table")
+data class PendingEventRetryEntity(
+    @PrimaryKey
+    val id: String,
+    val description: String,
+    val remindAt: Long,
+    val time: Long,
+    val title: String,
+    val action: AgendaItemAction
+
+)
```

Comment:
Just saving the id and action should be enough, since with that ID you can retrieve the actual event from the real DB. Saving duplicate data like this is very dangerous, because the real event could've already been updated, but not this entity here